selectel/resource_selectel_dbaas_postgresql_logical_replication_slot_v1.go

```diff
@@ -0,0 +1,187 @@
+package selectel
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/selectel/dbaas-go"
+)
+
+func resourceDBaaSPostgreSQLLogicalReplicationSlotV1() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceDBaaSPostgreSQLLogicalReplicationSlotV1Create,
+		ReadContext:   resourceDBaaSPostgreSQLLogicalReplicationSlotV1Read,
+		DeleteContext: resourceDBaaSPostgreSQLLogicalReplicationSlotV1Delete,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceDBaaSPostgreSQLLogicalReplicationSlotV1ImportState,
+		},
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(60 * time.Minute),
+			Update: schema.DefaultTimeout(60 * time.Minute),
+			Delete: schema.DefaultTimeout(60 * time.Minute),
+		},
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"project_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"region": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					ru1Region,
+					ru2Region,
+					ru3Region,
+					ru7Region,
+					ru8Region,
+					ru9Region,
+					nl1Region,
```

Comment:
According to [PR](https://github.com/selectel/terraform-provider-selectel/pull/217) maybe we need to add `uz1Region` ?

Reply:
Fixed. Also removed `nl1Region`